pipeline/backend/kubernetes/pod.go

```diff
@@ -97,10 +88,10 @@ func Pod(namespace string, step *types.Step) *v1.Pod {
 				Name:            podName(step),
 				Image:           step.Image,
 				ImagePullPolicy: pullPolicy,
-				Command:         command,
-				Args:            args,
+				Command:         step.Entrypoint,
+				Args:            step.Commands,
```

Comment:
can somebody test this?

-> so that it's possible to forward commands directly into kubernetes